web/src/views/repo/RepoWrapper.vue

```diff
@@ -17,7 +17,7 @@
       <a v-if="badgeUrl" :href="badgeUrl" target="_blank">
         <img :src="badgeUrl" />
       </a>
-      <IconButton :href="repo.forge_url" :title="$t('repo.open_in_forge')" :icon="forge ?? 'repo'" />
+      <IconButton :href="repo.forge_url" :title="$t('repo.open_in_forge')" :icon="forge ?? 'repo'" class="forge" />
```

Comment:
Do you want to use it for e2e test? In that case I would suggest

```suggestion
      <IconButton :href="repo.forge_url" :title="$t('repo.open_in_forge')" :icon="forge ?? 'repo'" data-test="forge" />
```

Reply:
No for custom styling on codeberg